src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -744,6 +753,86 @@ def load_ldlc_medication_effect(key: str, location: str) -> pd.DataFrame:
     return df
 
 
+def load_relative_risk_categorical_sbp(key: str, location: str) -> pd.DataFrame:
+    distributions = data_values.RELATIVE_RISK_SBP_ON_HEART_FAILURE_DISTRIBUTIONS
+    population_structure = load_population_structure(
+        data_keys.POPULATION.STRUCTURE, location
+    ).droplevel("location")
+
+    # define TMREL data
+    baseline_hf_rrs = pd.DataFrame(
+        1.0, index=population_structure.index, columns=ARTIFACT_COLUMNS
+    )
+    baseline_hf_rrs["parameter"] = "cat4"
+
+    # define exposed groups data
+    exposed_groups_rrs = []
+    for sbp_category, distribution in distributions:
+        rr_data = get_random_variable_draws(DRAW_COUNT, (sbp_category, distribution))
+        # relative risks of 1 for ages without heart failure (under 25)
+        under_25_data = pd.DataFrame(
+            1,
+            index=population_structure.query("age_start<=25").index,
+            columns=ARTIFACT_COLUMNS,
+        )
+
+        over_25_values = np.repeat(
+            [rr_data], len(population_structure.query("age_start>25")), axis=0
+        )
+        over_25_data = pd.DataFrame(
+            over_25_values,
+            index=population_structure.query("age_start>25").index,
```

Comment:
If you save this index in a variable before you set `over_25_values`, you only have to do this query once.